src/walker.rs

```diff
@@ -94,7 +103,21 @@ impl Walker {
       return Ok(Files::file(self.root, Bytes::from(root_metadata.len())));
     }
 
-    let filter = |entry: &walkdir::DirEntry| {
+    let mut file_infos = Vec::new();
+    let mut total_size = 0;
+
+    let mut walk_builder = WalkBuilder::new(&self.root);
+    walk_builder
+      .follow_links(self.follow_symlinks)
+      .standard_filters(false)
+      .hidden(!self.include_hidden);
```

Comment:
To keep this diff small, let's keep using the old implementation for skipping hidden files. I can rip out the old code in a follow-up diff.

Reply:
Unfortunately I don't think there's anything in `ignore` that directly matches `walkdir`'s [filter_entry](https://docs.rs/walkdir/2.3.1/walkdir/struct.FilterEntry.html#method.filter_entry).

Reply:
Ahh, gotcha. That's annoying. `ignore`'s `hidden` functionality doesn't skip files on MacOS that are hidden with a file attribute, but I think that's pretty minor. In the longer term, we can:

1. Make a PR to `ignore` to expose `filter_entry`, so that we can apply our own filtering.
2. Make a PR to `ignore` to respect the MacOS hidden file flag.

Until then, we can remove the test which checks that files hidden by flag aren't included. You can find it in `subcommand/torrent/create.rs` if you search for `chflags`.